_docs/reference/contribute/writing-a-new-topic.md

```diff
@@ -228,15 +228,14 @@ display a config file or a test file. To do so, you can't use normal markup and
 use direct HTML. For example:
 
 ```
-{% raw %}<pre data-src="{{home}}/repos/istio/BUILD"></pre>{% endraw %}
+{% raw %}<pre data-src="https://raw.githubusercontent.com/istio/istio/master/BUILD"></pre>{% endraw %}
```

Comment:
This won't work. The URL reference needs to be within the current site, otherwise the browser will block the fetch. This is why we have the repos/istio submodule within the repo.